assets/dev/js/editor/regions/navigator/elements/hooks/ui/document/elements/settings/navigator-render-indicators.js

```diff
@@ -0,0 +1,38 @@
+import After from 'elementor-api/modules/hooks/ui/after';
+
+export class NavigatorRenderIndicators extends After {
+	getCommand() {
+		return 'document/elements/settings';
+	}
+
+	getId() {
+		return 'navigator-render-indicators--/panel/editor/open';
+	}
+
+	getConditions( args ) {
+		return 'string' === typeof args.settings._title;
```

Comment:
Check if the navigator is open.

Reply:
Handled: https://github.com/elementor/elementor/pull/14264/commits/b2be23d45787ac398a1243da5b24230a36c44e0e